Assets/HoloToolkit/Input/Scripts/FocusedObjectMessageSender.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+using HoloToolkit.Unity;
+
+public class FocusedObjectMessageSender : MonoBehaviour
```

Comment:
Sorry to harp on this.
Feels like an overkill to have another script attached onto my game object to do something as simple as GazeManager.Instance.FocusedGameObject.SendMessage("FooBar");

I vote not having this script at all. And your change simply make the change to make focusedGameObject public.

@jwittner and @keveleigh please weigh in on your preference.